scripts/deploy.js

```diff
@@ -21,21 +21,11 @@ async function deployContract() {
   console.log("storage impl address is ", impl);
 
   const transaction = await implContract.populateTransaction.initialize(
```

Comment:
It would be nice if we have a gas profiling for `putBlob` to see if it achieves our optimization target. The profile should be something like this: https://github.com/ethstorage/storage-contracts-v1/pull/68#issuecomment-1916450325